### PR TITLE
Add/Issue-493: Serial publications

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

Yes, issue # 493 

### Please describe the goal of this pull request and the changes that were made.

Help users withs serial publications understand the best way to create new projects while saving all their custom work.

### CODE REVIEW

File: content/documentation/github.md [hunk 1]
```diff
@@ -63,6 +63,10 @@ If you are collaborating on a project, and one of your colleagues has already ho
 - To view a preview of a Quire project cloned from GitHub, you’ll need to first run the `quire install` command in your command-line shell. Then run `quire preview` and navigate to [http://localhost:1313/](http://localhost:1313/).
 {{< /q-class >}}
 
+{{< q-class "box tip" >}}
+- Learn how to take an existing Quire project in GitHub and use it as the basis for a new one (for a journal or other serial publication for instance) in our [*Create a New Quire Project from an Existing One*](/documentation/serial-publications/) section.
```
Comment (geealbers) [inline content/documentation/github.md:67]: Perhaps this tip should also be in this section: https://quire.getty.edu/documentation/getting-started/#copy-an-existing-project. Note in particular where it says at the end of the first paragraph "or if you wanted to use a previous Quire project as a template for a new one."

**Comment by Erin-Cecele** [comment]
@geealbers just pushed up a few edits. Let me know if it works. I'm not sure about the double tip in the Getting Started section.

**Comment by geealbers** [comment]
Yes, that double tip doesn't really work. I'd move the `cd` one somewhere else maybe? It seems less relevant here. Or maybe convert the GitHub one to a regular paragraph of text.

**Comment by Erin-Cecele** [comment]
@geealbers ok I moved the drag-and-drop tip to the tutorial section and pushed up a few more small changes. Let me know how that looks.

**Review by geealbers — APPROVED** [review]
Looks good to me. Thanks for getting everything woven in.

**Pull request merged**